reservations/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class Reservation(models.Model):
+    name         = models.CharField(max_length = 45)
+    phone_number = models.CharField(max_length = 45)
+    check_in     = models.DateField()
+    check_out    = models.DateField()
+    status       = models.ForeignKey('Status', on_delete = models.SET_NULL, null = True)
+    user         = models.ForeignKey('users.User', on_delete = models.CASCADE)
+    hotel        = models.ForeignKey('hotels.Hotel', on_delete = models.SET_NULL, null = True)
+    room_type    = models.ForeignKey('hotels.RoomType', on_delete = models.SET_NULL, null = True)
+    created_at   = models.DateField(auto_now_add = True)
```

Comment:
예약 변경을 대비해서 updated 도 추가해주세요.
```suggestion
    created_at   = models.DateTimeField(auto_now_add = True)
```

Reply:
Time 넣지 않으려고 일부러 DateField 로 했습니당. 저희는 날짜만 필요합니다. 저희 사이트는 예약 시간은 보여지지 않고 날짜만 보여지게 하려고 했습니다. Update는 추가 하겟습니당~~~

Reply:
승찬님 created_at 혹은 updated_at 과 같은 데이터가 저장되는 시점을 담을 때는 확장성을 고려하여 datefield 보다는 datetimefield 를 선호합니다.
실제로 서비스에서 연, 월, 일 만 표시된다 하더라도 table의 created_at 에 있는 데이터를 문자열로 가공하여 보내주는 것이기 때문에, 추후 갑작스럽게 연, 월, 일 외 적인 부분도 서비스에 노출시키겠다고 하는 경우 다시 migrate 를 해야하는 경우가 발생할 수 있겠죠!

굳이 확장성을 고려하지 않고 datefield 로 진행하셔도 무방합니다!